lib/create-space-api.js

```diff
@@ -1607,6 +1657,8 @@ export default function createSpaceApi ({
     createUiExtension: createUiExtension,
     createUiExtensionWithId: createUiExtensionWithId,
     getEntrySnapshots: getEntrySnapshots,
-    getContentTypeSnapshots: getContentTypeSnapshots
+    getContentTypeSnapshots: getContentTypeSnapshots,
+    getEnvironmentAlias,
```

Comment:
style-wise, is it weird to use the ES6 convention here and not everywhere else? should I change all of them? should I make these last two match the others?

Reply:
Might be best to change all of them.